client/src/featureform/client.py

```
@@ -198,8 +204,75 @@ def _iceberg_dataframe(
+
+        print("Building ticket...")
+        ticket = flight.Ticket(json.dumps(ticket_data).encode("utf-8"))
+
+        try:
+            print("Attempting to fetch data via Arrow Flight from Go Proxy...")
+            stream = flight_client.do_get(ticket)
+            reader = stream.read_all()
+            df = reader.to_pandas()
+            return df
+        except flight.FlightError as e:
+            raise RuntimeError(f"Failed to fetch data from Go proxy: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)